packages/hdwallet-provider/.npmignore

```diff
@@ -1,2 +1,7 @@
 .idea
 .vscode
+test
+.travis.yml
```

Comment:
Oh, there'a  `.travis.yml` in here?!!

Are you open to removing it as part of this PR?

Reply:
Why do we need .travis.yml in the final build of the package?

Reply:
Ah, I miscommunicated. I was referring to the pkg itself.

I didn't know there was a `.travis.yml` file in the pkg. We no longer use it (probably leftover from before this pkg was a part of the monorepo). Are you open to removing the `travis.yml` from this pkg as part of the PR?

Reply:
Yeah sure

Reply:
Done